src/main/java/xyz/aadev/generitech/GeneriTechTabs.java

```diff
@@ -76,7 +76,7 @@ public String getTabLabel() {
         }
     };
 
-    public static final CreativeTabs FLUID = new CreativeTabs(Reference.MOD_ID) {
+    /*public static final CreativeTabs FLUID = new CreativeTabs(Reference.MOD_ID) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3ACommentedOutCodeLine)